src/vm/clrvarargs.cpp

```diff
@@ -33,10 +33,14 @@ void VARARGS::MarshalToManagedVaList(va_list va, VARARGS *dataout)
 {
     WRAPPER_NO_CONTRACT
 
+#if !(defined(_TARGET_ARM_) && defined(PLATFORM_UNIX))
```

Comment:
Support for managed varargs (`System.ArgIterator` and friends) is not part of .NET Core profile. No need to worry about it for Linux ARM port. It does not work on Linux AMD64 either as @janvorli points out.